### PR TITLE
Utilize generic prefab for objects

### PR DESCRIPTION
**Pull request by vchadha**
Changed systems to use generic prefabs
Reorganized constants a little too

### CODE REVIEW

File: Assets/Scripts/Constants/Textures.cs [hunk 1]
```diff
@@ -0,0 +1,56 @@
+using System.Collections;
```
Comment (vchadha) [inline Assets/Scripts/Constants/Textures.cs:1]: I totally didn't mean to commit this... I was messing around for a bit, but this is future work...
I could remove it, but I'd have to recommit ;-;

File: Assets/Scripts/Constants/Constants.cs [hunk 2]
```diff
@@ -0,0 +1,5 @@
+public class Constants
```
Comment (justinro-underscore) [inline Assets/Scripts/Constants/Constants.cs:1]: OOooo you know how much I love a Constants file

File: Assets/Scripts/Constants/Textures.cs (hunk 1 above)
Comment (justinro-underscore) [reply Assets/Scripts/Constants/Textures.cs:1]: All good! If it's future work, then let's keep it in

File: Assets/Scripts/Constants/Types.cs [hunk 3]
```diff
@@ -1,5 +1,16 @@
-public class Constants
+public class Types
```
Comment (justinro-underscore) [inline Assets/Scripts/Constants/Types.cs:1]: Optional: Hm, I don't know if `Types` is the best name for this. Maybe `PizzaTypes` or `PizzaIngredients` or something?

**Review by justinro-underscore — APPROVED** [review]
Love cutting down on our prefab count!

**Review by justinro-underscore — APPROVED** [review]

**Pull request merged**